File: src/main/java/icu/takeneko/omms/client/data/system/ProcessorInfo.java
package icu.takeneko.omms.client.data.system;

import lombok.AllArgsConstructor;
import lombok.Getter;

@Getter
@AllArgsConstructor
public class ProcessorInfo {
    private int physicalCPUCount;
    private int logicalProcessorCount;
    private String processorName;
    private double cpuLoadAvg;
    private String processorId;
    private double cpuTemp;

}

File: src/main/java/icu/takeneko/omms/client/data/system/ProcessorInfo.kt
package icu.takeneko.omms.client.data.system

data class ProcessorInfo(
    var physicalCPUCount: Int = 0,
    var logicalProcessorCount: Int = 0,
    var processorName: String = "",
    var cpuLoadAvg: Double = 0.0,
    var processorId: String = "",
    var cpuTemp: Double = 0.0,
)
